skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp

```diff
@@ -6,6 +6,7 @@
 #include <functional>
 #include <sstream>
 #include <stdexcept>
+#include <exception>
```

Comment:
`runtime_error` not belongs to this header:
https://en.cppreference.com/w/cpp/error/runtime_error